pom.xml

```diff
@@ -47,6 +47,7 @@
     <version.assertj-core>3.8.0</version.assertj-core>
     <version.system-rules>1.16.1</version.system-rules>
     <version.javassist>3.21.0-GA</version.javassist>
+    <version.fast-classpath-scanner>2.7.4</version.fast-classpath-scanner>
```

Comment:
Really light project for scanning classpath. I think we can use more and more features it offers.